### DISCUSSION TITLE
Duplicate Directories entries in Midnight Commander (mc)

### DISCUSSION THREAD

**Discussion by MarcelSchaible** [trapexit/mergerfs#993] [Q&A] [answered]
I am using midnight commander to copy files and directories on my mergerfs filesystem around. MC is listing all directories strangely twice. Is there an mount option to prevent this. Since I am lost with this problem any hint or suggestion is welcome.

**Comment by trapexit** [comment]
Can you provide some details? Like the version of mergerfs?

**Comment by mighq** [comment]
This happens to me too. Also duplicates the files. All files are shown twice.

versions:
* mc: 3:4.8.26-1.1
* mergerfs: 2.31.0-1
* OS: debian 11.4 (arm64)

**Reply by mighq** [reply]
Also happens when I try the simplest case with 1 branch only, 1 file in the tree. In MC I see the file twice. For other tools I see just 1 file as expected.

**Comment by mighq** [comment]
Seems like it is related to this issue
https://github.com/trapexit/mergerfs/issues/877

**Reply by trapexit** [reply]
As asked in the docs please use the latest version of mergerfs before reporting issues. 2.31 is quite old.

**Comment by petras-sukys** [comment, marked as answer]
Yup, updated on Debian 11 with .deb from github releases, rebooted (probably remounting would have been enough), and everything works fine :)

**Reply by trapexit** [reply]
Glad to hear. Always check the latest release. Your OS, unless you're using a rolling release distro, won't update for you.
